.circleci/config.yml

```diff
@@ -262,6 +264,20 @@ jobs:
 
       - store_artifacts:
           path: conda_env_dump.log  # This is the list of modules in conda and pip
+
+      - run:
+          name: Run HITL functionality
+          no_output_timeout: 60m
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat; cd habitat-lab
+              ln -s ../habitat-sim/data data
+              python -m habitat_sim.utils.datasets_download --uids hab3-episodes habitat_humanoids hab_spot_arm ycb hssd-hab --data-path data/ --no-replace --no-prune
```

Comment:
We can't download hssd-hab in a CI test--it is quite large (~20GB). Talk to @aclegg3 about a CI-friendly version of this dataset.

Reply:
https://huggingface.co/datasets/ai-habitat/hab3_bench_assets

Reply:
FYI @henrysamer we still need to change this PR to not download hssd-hab. I realize the test is passing, but we shouldn't have CI doing such a large download.

Reply:
Might need to understand why dataset episodes aren't being found here